lib/clockwise_import.rb

```diff
@@ -0,0 +1,211 @@
+require 'csv'
+
+class ClockwiseImporter
+  def initialize(csv_path:, tenant:)
+    @csv_path = csv_path
+    @tenant = tenant
+    Apartment::Tenant.switch @tenant
+  end
+
+  def import!
+    Entry.transaction do
+      CSV.foreach(@csv_path,
+                  headers: true,
+                  quote_char: '"',
+                  col_sep: ';') do |r|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.